### DISCUSSION TITLE
noise from esp32-a1s

### DISCUSSION THREAD

**Discussion by seancmalloy** [sle118/squeezelite-esp32#375] [General]
I'm trying to use an external amp via audio jack on esp32-a1s flashed with squeezelite-esp32 (latest)  but I'm getting a lot of noise on speakers from this board. I've tried using linear regularator for amp and for esp32-a1s board but for whatever reason it won't properly boot up so is currently on buck convertor at 5v. Has anyone delt with this issue? Also, does anyone know if the esp32-a1s board speaker output shares a ground; can I connect to 3 pin audio input amp by combining ground wires?

**Comment by Renber78** [comment]
Good morning,
Unfortunately you don't give enough information to receive help...
You are talking about ESP32-A1S, but are you using a ready-made audio card or is it just the chip? Which DAC do you use?

Generally:
- the DAC power supply must be as clean as possible and different from that of the ESP
- the shortest I2S line and without series resistance (or very low)
- Avoid ground loops (star wiring with a single starting point)
- Your power supply must be isolated (do not use the USB of a PC for example)

**Comment by seancmalloy** [comment]
Sorry. i am referring to ESP32-A1S audio kit / dev board. i am powering it with the 2-pin connector, not usb port. I tried to use lm317 linear regulator after the main step down to "clean up" the power  but the board would not properly boot up. so i tried a better quality 24v - 5v step down after the main 110v - 24v step down, which improved it a little  not much. i am trying to use the built in 3.5 audio jack / line out for external 2.1 amp.

![ESP32-A1S](https://github.com/sle118/squeezelite-esp32/assets/23019226/589a30b1-864b-46cf-b6b9-28d1d44ea92f)
